x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/components/constants.ts

```diff
@@ -10,10 +10,4 @@ import type { DiffableAllFields } from '../../../../../../../common/api/detectio
 type NonEditableFields = Readonly<Set<keyof DiffableAllFields>>;
 
 /* These fields are not visible in the comparison UI and are not editable */
-export const HIDDEN_FIELDS: NonEditableFields = new Set([
-  'alert_suppression',
```

Comment:
@nikitaindik `alert_suppression` actually needs to have be displayed in the diff and be able to be customized by the user. Can you make a note of that for the work that you and Maxim are currently doing?

Reply:
Will do.

Actually, can you please remove this whole file since these constants are not used anywhere. It's a duplicate of [this](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detection_engine/rule_management/model/prebuilt_rule_upgrade/fields.ts#L64) constant. I thought I removed it but it looks like it got resurrected during a merge.

Also you can now remove `author` and `license` from [NON_UPGRADEABLE_DIFFABLE_FIELDS](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/detection_engine/rule_management/model/prebuilt_rule_upgrade/fields.ts#L64).